specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -1589,6 +1595,10 @@
             "name": "DynamicCompressionEnabled",
             "modelAsString": true
           }
+        },
+        "cacheDuration": {
+          "description": "The duration for which the content needs to be cached. Allowed format is [d.]hh:mm:ss",
+          "type": "string"
```

Comment:
Is this an IS8601 duration, if so, we should mark it as format "duration".  If not, why isn't it?

Reply:
From RP perspective all we care is that the duration that we accept can't go over a year (31622400 seconds).
What are the implications from SDK standpoint of marking it as "string" versus "duration" ?

---
In reply to: [382810961](https://github.com/Azure/azure-rest-api-specs/pull/8455#discussion_r382810961) [](ancestors = 382810961)

Reply:
If we use the standard, there is support in multiple languages for representing as a local type representing a duration (for example, as a TimeSpan in .Net).  It would be good to document the limits (you imply them, but do not state that hh < 24

Reply:
Sounds good, added "format": "duration" field and updated examples.